File: qe-selenium/qe-selenium-python/Activity6.py
from selenium.webdriver.common.by import By
from selenium import webdriver

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls/")
    print(driver.title)
    
    print(driver.find_element(By.XPATH,"//*[@id=\"checkbox\"]").is_selected())
    driver.find_element(By.XPATH,"//*[@id=\"checkbox\"]").click()
    print(driver.find_element(By.XPATH,"//*[@id=\"checkbox\"]").is_selected())
    driver.quit

File: qe-selenium/qe-selenium-java/src/main/java/com/demos/Activity6.java
package com.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity6 {
    public static void main(String[] args) {
        FirefoxDriver driver =new FirefoxDriver();
        driver.get("https://training-support.net/webelements/dynamic-controls/");
        System.out.println(driver.findElement(By.xpath("//*[@id=\"checkbox\"]")).isDisplayed());
        driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/section[1]/button")).click();
        System.out.println(driver.findElement(By.xpath("//*[@id=\"checkbox\"]")).isDisplayed());
        driver.quit();


    }
}
